schedule/yast/firstboot/yast2_firstboot_custom-sle.yaml

```diff
@@ -24,5 +25,8 @@ test_data:
         - firstboot_hostname
         - firstboot_timezone
         - firstboot_user
-        - firstboot_root
     custom_control_file: "yast2/firstboot/firstboot_custom-sle.xml"
+    custom_license: "yast2/firstboot/license.txt"
```

Comment:
we could have these paths only in 'enable_y2_firstboot', if those are moved/renamed in the future in /data, we just need to update it in one place.

Reply:
Good point, done.